### PR TITLE
Bring hyb2onc2isis up-to-date with dev

### PR DESCRIPTION
**Pull request by scsides**
Brought the code up to date with dev, and fixed tests

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read and agree to abide by the [Code of Conduct](https://usgs-astrogeology.github.io/code/)
- [X] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have added tests to cover my changes.
- [X] All new and existing tests passed.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [X] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

File: isis/src/hayabusa2/apps/hyb2onc2isis/Hayabusa2OncBandBinUpdated.trn [hunk 1]
```diff
@@ -0,0 +1,75 @@
+#
+# This translation file is used to produce the ISIS 3 BandBin label group
+# for the Hayabusa 2 ONC-T camera.
+# Translation = (left,right) convert the input "right" to "left"
+#
+
+Group = FilterNumber
+  Auto
+  Optional
```
Comment (Kelvinrr) [inline isis/src/hayabusa2/apps/hyb2onc2isis/Hayabusa2OncBandBinUpdated.trn:9]: Is this just renamed?

File: isis/src/hayabusa2/apps/hyb2onc2isis/Hayabusa2OncArchiveUpdated.trn [hunk 2]
```diff
@@ -0,0 +1,107 @@
+#
+# This translation file is used to produce the ISIS 3 Archive label group
+# for the Hayabusa 2 ONC-W1, ONC-W2, and ONC-T cameras.
+#
+
+Group = ProducerId
+  Auto
```
Comment (Kelvinrr) [inline isis/src/hayabusa2/apps/hyb2onc2isis/Hayabusa2OncArchiveUpdated.trn:7]: Same for this one, you can probably extend this to all the "updated" files

File: isis/scripts/isisStartup.csh [hunk 3]
```diff
@@ -0,0 +1,37 @@
+#!/bin/csh
+################################################################################
+# This file should be sourced within your current shell using the "source"
+# command.
+#
+# On the command line type:
+# > setenv ISISROOT ????
+# > source isis3Startup.csh
+#
+# Replace the "????" in the above command line with the path you installed
```
Comment (Kelvinrr) [inline isis/scripts/isisStartup.csh]: Should we merge https://github.com/USGS-Astrogeology/ISIS3/pull/3927 before merging this?

**Review by Kelvinrr — COMMENTED** [review]
Just some questions in line.

File: isis/src/hayabusa2/apps/hyb2onc2isis/Hayabusa2OncBandBinUpdated.trn (hunk 1 above)
Comment (scsides) [reply isis/src/hayabusa2/apps/hyb2onc2isis/Hayabusa2OncBandBinUpdated.trn:9]: If you mean the "Updated" file name. The program works for both old format and new format files. They changed significantly, so I used different translation files.

File: isis/src/hayabusa2/apps/hyb2onc2isis/Hayabusa2OncArchiveUpdated.trn (hunk 2 above)
Comment (scsides) [reply isis/src/hayabusa2/apps/hyb2onc2isis/Hayabusa2OncArchiveUpdated.trn:7]: ditto

File: isis/scripts/isisStartup.csh (hunk 3 above)
Comment (scsides) [reply isis/scripts/isisStartup.csh]: Ugg, somehow I forgot to change branches. I'll fix this before.

**Review by kaitlyndlee — APPROVED** [review]
Looks good

**Pull request merged**